sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.aest.controller.assessOfferingChangeRequest.e2e-spec.ts

```diff
@@ -305,6 +310,119 @@ describe("EducationProgramOfferingAESTController(e2e)-assessOfferingChangeReques
     },
   );
 
+  it(
+    "Should determine the offering change for an offering with one or more applications not in completed status " +
+      "when the offering change is approved.",
+    async () => {
+      // Arrange
+
+      const notCompletedApplicationStatus = [
+        ApplicationStatus.InProgress,
+        ApplicationStatus.Assessment,
+        ApplicationStatus.Enrolment,
+      ];
+      const applications: Application[] = [];
```

Comment:
Minor. it can be applicationId array to use directly in query.

Reply:
Done.